hummingbot/client/config/config_var.py

```diff
@@ -13,10 +13,10 @@ def __init__(self,
                  type_str: str = "str",
                  # Whether this config will be prompted during the setup process
                  required_if: Callable = lambda: True,
-                 validator: Callable = lambda *args: True,
+                 validator: Callable = lambda *args: None,
```

Comment:
We should probably define the types of `required_if`, `on_validated` and `validator` more clearly, preferably with some type defs.

e.g.

```
Validator = Callable[[str], Optional[str]]
...
...
    validator: Validator = lambda *args: None,
```

Reply:
Ok, I have included this in the PR https://github.com/CoinAlpha/hummingbot/pull/1615